.gitignore

```diff
@@ -12,7 +12,6 @@ logs/
 .idea/
 .npm/
 .vagrant/
-/.vscode/
```

Comment:
removing this may cause trouble as people will commit their settings, won't it?

Reply:
yes, I'll ignore the file back, and with your suggestion to run the tests on  `**/nhost/config.yaml` in the repo, we'll be sure everything is configured correctly